ELCCpv1.py

```diff
@@ -33,3 +29,16 @@
    }
 """
 """Example: {"UserID":"009mx87m543567nqnnbvcdretyupam3mmmwiqw4r","module": "Solarworld Sunmodule Plus SW 265 mono black SW-01-6023US","inverter":"Enphase M250 Microinverter 800-00181-r06","data-logger":"","pyranometer":"","windsensor":"","rainsensor":"","Web_layer_API":"","Size_kW":"3.975","lat":"51.678N","long":"0.301E","Comment":"any comment up to 40char, RPI 2b, static, solar powered","generation":"2017-04-20-14:10:25-2017-04-20-14:15:27","MWh":"9.046878"}"""
```

Comment:
Could you please update the example to reflect the new format?

Reply:
Sure I will take a look